docker-compose-dependencies.yml

```diff
@@ -0,0 +1,221 @@
+      - microserviceKeys_em_npa_app=AAAAAAAAAAAAAAAA
+      #      logging env vars
+      - JSON_CONSOLE_PRETTY_PRINT=false
+      - REFORM_SERVICE_TYPE=java
+      - REFORM_SERVICE_NAME=service-auth-provider-api
+      - REFORM_TEAM=rpe
+      - REFORM_ENVIRONMENT=docker
+#      healthcheck env vars
+      - PACKAGES_ENVIRONMENT=docker
+      - PACKAGES_PROJECT=rpe
+      - PACKAGES_NAME=service-auth-provider-api
+      - PACKAGES_VERSION=unkown
+      - TESTING_SUPPORT_ENABLED=true
+    ports:
+      - 4502:8489
+
+  annotation-api:
+    image: hmctspublic.azurecr.io/em/anno:latest
+    container_name: npa-anno-api
+    environment:
+      - SPRING_DATASOURCE_URL=jdbc:postgresql://shared-database:5432/anno
+      - SPRING_DATASOURCE_USERNAME=shared
+      - SPRING_DATASOURCE_PASSWORD=shared
+      - S2S_BASE_URI=http://service-auth-provider-api:8489
+      - IDAM_API_BASE_URI=http://idam-api:5000
+      - OPEN_ID_API_BASE_URI=http://idam-api:5000/o
+      - ENABLE_LIQUIBASE=true
+    depends_on:
+    - shared-database
+    ports:
+    - 4623:8080
+
+  idam-api:
+    image: hmctspublic.azurecr.io/idam/api:stable
+    container_name: npa-idam-api
+    healthcheck:
+      interval: 10s
+      timeout: 10s
+      retries: 10
+    environment:
+      - JAVA_TOOL_OPTIONS=-Xms128m -Xmx512m -XX:MaxPermSize=512m
+      - SPRING_PROFILES_ACTIVE=local
+      - CACHING_ENABLED=false
+      - SPRING_DATA_ELASTICSEARCH_PROPERTIES_PATH_HOME=/tmp/es
+      - TESTING_SUPPORT_ENABLED=true
+      - IDAMHEALTHCHECK_AM_ENABLED=false
+      - IDAMHEALTHCHECK_IDM_ENABLED=false
+      - STRATEGIC_ADMIN_URL=http://idam-web-admin:8082
+      - STRATEGIC_WEBPUBLIC_URL=http://idam-web-public:9002
+      - SPRING_DATASOURCE_URL=jdbc:postgresql://shared-db:5432/openidm?currentSchema=fridam
+      - SPRING_DATASOURCE_USERNAME=openidm
+      - SPRING_DATASOURCE_PASSWORD=openidm
+      - SECURITY_OAUTH2_CLIENT_CLIENTSECRET=password
+      - SECURITY_OAUTH2_CLIENT_PRE_ESTABLISHED_REDIRECT_URI=http://idam-web-public:9002/login
+      - SECURITY_OAUTH2_CLIENT_REGISTERED_REDIRECT_URI=http://idam-web-public:9002/login
+      - IDAM_SPI_FORGEROCK_AM_ROOT=http://fr-am:8080/openam
+      - IDAM_SPI_FORGEROCK_AM_TOPLEVELHOST=fr-am
+      - IDAM_SPI_FORGEROCK_AM_USERNAME=amadmin
+      - IDAM_SPI_FORGEROCK_AM_PASSWORD=Pa55word11
+      - IDAM_SPI_FORGEROCK_AM_JWKSURIFOROAUTH2CLIENTS=http://fr-am:8080/openam/oauth2/hmcts/connect/jwk_uri
+      - IDAM_SPI_FORGEROCK_IDM_ROOT=http://fr-idm:18080/openidm
+      - IDAM_SPI_FORGEROCK_IDM_USERNAME=openidm-admin
+      - IDAM_SPI_FORGEROCK_IDM_PASSWORD=openidm-admin
+      - IDAM_SPI_FORGEROCK_IDM_PIN_DEFAULTPASSWORD=BlaBlaBlackSh33p
+      - IDAM_API_OAUTH2_CLIENT_CLIENT_SECRETS_WEBSHOW=AAAAAAAAAAAAAAAA
+      - IDAM_WEBSHOW_WHITELIST=http://localhost:8080/oauth2redirect
+      - REFORM_SERVICE_NAME=idam-api
+      - REFORM_TEAM=idam
+      - REFORM_ENVIRONMENT=local
+    ports:
+      - 5000:5000
+
+  idam-web-public:
```

Comment:
do we need these idam-web containers, given that we do everything we need through idam-api? it's just takes up more resources on our machines